docs/dev/ci/github_actions/runners.md

```diff
@@ -65,6 +72,52 @@ The available configurations are:
 
 * `*` - Not specified in the group name
 
+### Dedicated GPU Runners
+
+As of 24/05/2024,
+18 runners with GPU capabilities are available to all repositories in the OpenVINO organisation's GitHub Actions.
```

Comment:
I would add something like: "Both integrated GPUs (iGPU) and discrete GPUs (dGPU) are available". so iGPU/dGPU would be explained when mentioned for the first time.